lib/parse.sh

```
@@ -1,5 +1,7 @@
+source lib/ui.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)